README.md

```diff
@@ -1,23 +1,31 @@
 Chitter Challenge
 =================
 
-* Feel free to use Google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
 
-Challenge:
--------
-
-As usual please start by forking this repo.
+In order to user this web-app please follow these instructions:
 
-We are going to write a small Twitter clone that will allow the users to post messages to a public stream.
+* Clone this repo
+* Run 'bundle' to install the required gems
+* Connect to psql
+* Create the database using the psql command CREATE DATABASE chitter;
+* Connect to the database using the pqsl command \c chitter;
+* Run the query saved in the file db/01_create_peeps_table.sql
```

Comment:
This file should also note about setting up the test database. I see that table is called peeps, not bookmark.